common/src/main/java/com/hedera/block/common/constants/ErrorMessageConstants.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.block.common.constants;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+
+/** A class that holds common error message {@link String} literals that are used across projects */
```

Comment:
we could remove the "@link String` since I don't think it adds that much value.

Reply:
could agree that there is not much added value for the class `String`

Reply:
removed